### ISSUE TITLE
Add filter capabilities to protocol grid view (Trac #70)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#74]
Add right-click menu (e.g., 'Select columns...') to protocol grid view to allow user to select which columns to show in the protocol grid view.

Migrated from http://microfluidics.utoronto.ca/ticket/70
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Add right-click menu (e.g., 'Select columns...') to protocol grid view to allow user to select which columns to show in the protocol grid view.", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Add filter capabilities to protocol grid view", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-14T15:55:22", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120315T05:16:14: ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120330T19:14:03: christian@fobel.net commented:

> commit 18959bf31771b5b2679f28ac4fd35728e5f3d56d
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Mar 28 21:18:04 2012 -0400
> 
>     Add enabled_fields property on ProtocolGridCont...
>     
>     Add enabled_fields property on ProtocolGridController.  This property contains
>     a dictionary mapping a plugin name to the names of the attributes that should
>     be shown in the protocol grid view.  If the dictionary is empty (default), all
>     plugin field attributes are included in the grid view.
>     
>     See #74.

**Comment by cfobel** [comment]
Trac update at 20120403T07:22:32: 
* christian@fobel.net commented:

> commit 14c04f1918dc26b4311c5586eec5e035fefdaddd
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Apr 3 03:16:02 2012 -0400
> 
>     Add filter capabilities to protocol grid view
>     
>     Fixes #74.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
